.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+version: 2
+jobs:
+  build:
+    machine:
+      image: circleci/classic:201710-02
```

Comment:
I suspect we can probably just use the `python:3.7` docker image, but if this works for now, that's fine.